packages/tab-scroller/index.tsx

```diff
@@ -151,14 +151,40 @@ export default class TabScroller extends React.Component<
       getScrollContentOffsetWidth: this.getScrollContentWidth,
       getScrollAreaOffsetWidth: () =>
         this.areaElement.current ? this.areaElement.current.offsetWidth : 0,
-      computeScrollAreaClientRect: () =>
-        this.areaElement.current ?
-          this.areaElement.current.getBoundingClientRect() :
-          new ClientRect(),
-      computeScrollContentClientRect: () =>
-        this.contentElement.current ?
-          this.contentElement.current.getBoundingClientRect() :
-          new ClientRect(),
+      computeScrollAreaClientRect: () => {
+        if (!this.areaElement.current) {
+          // new DOMRect is not IE11 compatible
+          const defaultDOMRect = {
```

Comment:
Could you make this a variable that both `computeScrollAreaClientRect` and `computeScrollContentClientRect` reference, instead of defining it twice?

Reply:
I created a function `getBoundingClientRectOf` within `tab-scroller`.

I considered a shared variable (either within the class or the `adapter`) but if code happened to modify the returned object, then it would result in these functions no longer returning the expected default object.